files/build_templates/sonic_debian_extension.j2

```diff
@@ -216,6 +216,11 @@ sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog.conf.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_
 sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog.d/* $FILESYSTEM_ROOT/etc/rsyslog.d/
 echo "rsyslog-config.service" | sudo tee -a $GENERATED_SERVICE_FILE
 
+#copy syslog override files
```

Comment:
>#copy [](start = 0, length = 5)

Better to follow the code style in the same file. We may need more space between # and character to have better breath.

Reply:
Accepted